File: Java/Code/Baju.java
public class Baju extends Aksesoris{
    private String untuk;
    private String size;
    private String merk;

    public Baju() {
    }

    public Baju(String id, String namaProduk, int hargaProduk, int stokProduk, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, namaProduk, hargaProduk, stokProduk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return this.untuk;
    }

    public String getSize() {
        return this.size;
    }

    public String getMerk() {
        return this.merk;
    }
}

File: Python/Code/Baju.py
from Aksesoris import Aksesoris

class Baju (Aksesoris) :
    __untuk = ""
    __size = ""
    __merk = ""

    def __init__(self, ID, nama_produk, harga_produk, stok_produk, jenis, bahan, warna, untuk, size, merk):
        super().__init__(ID, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk

    def getUntuk(self) :
        return self.__untuk
    
    def getSize(self) :
        return self.__size
    
    def getMerk(self) :
        return self.__merk